src/Bundle/test/config/services_test_grids_as_service.yaml

```diff
@@ -2,6 +2,8 @@ services:
     _defaults:
         autowire: true
         autoconfigure: true
+        bind:
+            $authorClass: '%app.model.author.class%'
```

Comment:
Perhaps it could be improved with ResourceBundle to create such binds out-of-the-box

Reply:
Yes I had this same idea during development.